h/static/scripts/controllers.coffee

```diff
@@ -225,12 +152,44 @@ class AnnotationViewerController
     streamer.send({filter: streamfilter.getFilter()})
 
 class ViewerController
-  this.$inject = ['$scope', 'annotator']
-  constructor:   ( $scope,   annotator ) ->
+  this.$inject = [
+    '$scope', '$route',
+    'annotator', 'auth', 'flash', 'streamer', 'streamfilter', 'store'
+  ]
+  constructor:   (
+     $scope,   $route,
+     annotator,   auth,   flash,   streamer,   streamfilter,   store
+  ) ->
     # Tells the view that these annotations are embedded into the owner doc
     $scope.isEmbedded = true
     $scope.isStream = true
 
+    loaded = []
+
+    loadAnnotations = ->
```

Comment:
This feels like it's in the wrong place. I'd like to see the view concern itself only with adding properties to Scope and dealing with UI events. Lets extract this into something  like the Hypothesis class?